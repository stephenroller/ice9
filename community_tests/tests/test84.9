# = with ints and bools									succeed

write ? (1 = 1);
write ? (true = false);
write ? (true = (true = false));
