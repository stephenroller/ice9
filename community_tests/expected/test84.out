Loading file: generated/test84.tm
1 
0 
0 
Number of instructions executed = 60
